tsqc/pages/mutations.py

```diff
@@ -32,15 +33,16 @@ def make_hist_panel(tsm, log_y):
     return pn.Row(overall_site_hist, overall_node_hist)
 
 
-def page(tsm):
-    hv.extension("bokeh")
+def make_scatter_panel(log_y, tsm):
     plot_width = 1000
-    log_y_checkbox = pn.widgets.Checkbox(
-        name="Log y-axis of Mutations per site/node plots", value=False
-    )
+    y_dim = "time"
+    if log_y is True:
```

Comment:
Tiny nitpick here, better to use 'if log_y' rather than comparing for identity with True